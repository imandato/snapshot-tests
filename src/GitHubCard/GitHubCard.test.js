import { render, screen } from '@testing-library/react';
import GithubCard from './GitHubCard';
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
  const tree = renderer.create(<GithubCard/>).toJSON()
  expect(tree).toMatchSnapshot()
})