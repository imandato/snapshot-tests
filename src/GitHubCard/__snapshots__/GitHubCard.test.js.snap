// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="Chester.jpeg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      imandato
    </div>
    <p
      className="card-text"
    >
      Github description - adding on
    </p>
  </div>
</div>
`;
